### PR TITLE
Task 6 complete

### PR DESCRIPTION
**Pull request by Tomoneill98**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -191,13 +189,30 @@ describe("GET - /api/articles/:article_id/comments", () => {
...
+  it("GET - look for nonsense - Returns an error", () => {
+    return request(app)
+      .get("/api/articles/nonsense/comments")
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("Error - Invalid ID");
+      });
+  });
+  it("GET - look for an article with no comments - Returns a message saying that", () => {
+    return request(app)
```
Comment (EddTheDuck) [inline __tests__/app.test.js:211]: Returns a message saying what exactly?

Comment (EddTheDuck) [reply __tests__/app.test.js:211]: and what are we actually getting back ?

**Review by EddTheDuck — APPROVED** [review]
Just a little comment but otherwise happy for you to merge.

File: __tests__/app.test.js (hunk 1 above)
Comment (Tomoneill98) [reply __tests__/app.test.js:211]: will do those now, thank you!

**Pull request merged**